llvm/tools/dsymutil/Options.td

```diff
@@ -218,3 +218,16 @@ def dsym_search_path: Separate<["-", "--"], "D">,
   MetaVarName<"<path>">,
   HelpText<"Specify a directory that contain dSYM files to search for.">,
   Group<grp_general>;
+
+def cas : Separate<["-", "--"], "cas">,
+  MetaVarName<"<path>">,
+  HelpText<"Specify CAS directory to load CAS references from.">,
```

Comment:
Does this imply that there can only be one CAS per build?
Is that realistic? What if people use static archives and combine objects from different projects?

Reply:
Yes, it has to be one CAS per build. It is a requirement for module imports and there is no way you can configure otherwise.

If you have multiple builds and uses some products from other build, the second CAS has to ingest everything from previous builds.